components/shared/organisms/Header/UserNavigation.tsx

```diff
@@ -14,15 +14,15 @@ interface Props extends IUserNavigation {
 
 const UserNavigation = ({ user, links, actions }: Props) => {
   const [isOpen, setIsOpen] = useState(false);
-  const wrapperRef = useRef<any>(null);
+  const wrapperRef = useRef<HTMLDivElement>(null);
 
   const toggleDropdown = useCallback(() => {
     setIsOpen(!isOpen);
   }, [isOpen]);
 
   useEffect(() => {
-    const handleOutsideClick = (event: any) => {
-      if (wrapperRef?.current && wrapperRef?.current?.contains(event.target)) {
+    const handleOutsideClick = (event: MouseEvent) => {
+      if (wrapperRef?.current && event.target instanceof Node && wrapperRef?.current?.contains(event.target)) {
```

Comment:
also. i think we can move this function from useEffect body (look toggleDropdown)

and dont need to add dependency in array at useEffect. 

also. can remove toggleDropdown from 40 line.

Reply:
I don't think its a good idea to move function from useEffect according to https://reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies

Reply:
yeah. thank u! dont need really.